### ISSUE TITLE
MultiSelect not submiting payload inside a form

### ISSUE THREAD

**Issue by Adam1901** [carbon-design-system/carbon-components-svelte#1742]
When submiting a form, the multi select values are not been serialised into the payload

![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/16060576/24b5732e-c98e-4258-aedc-64b1c2e2d72c)

```
<script>
    import {Button, Form, MultiSelect, TextInput} from "carbon-components-svelte";
    import {enhance} from '$app/forms';
</script>

<Form method="post">
    <TextInput name="text"></TextInput>
    <MultiSelect
            id="items"
            name="items"
            titleText="Contact"
            label="Select contact methods..."
            items={[
    { id: "0", text: "Slack" },
    { id: "1", text: "Email" },
    { id: "2", text: "Fax" },
  ]}
    />
    <Button type="submit">Submit</Button>
</Form>
```

**Comment by Whidix** [comment]
Hello, I have the same issue. It would be good to have the possibility to submit it with selectedIds as value.

**Comment by theetrain** [comment]
This has been fixed in [v0.81.2](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.81.2).

**Comment by gschurck** [comment]
> This has been fixed in [v0.81.2](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.81.2).

@theetrain Why not using a hidden select multiple html form ? It could be easier for form validation with Zod and Superform to get an array of value when submitting form, instead of having some checkboxes.
